installers/image/bin/pgo-deploy.sh

```diff
@@ -11,34 +11,11 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-# Operator Defaults
-export BACKREST_PORT=${BACKREST_PORT:-2022}
-export CRUNCHY_DEBUG=${CRUNCHY_DEBUG:-false}
-export DELETE_METRICS_NAMESPACE=${DELETE_METRICS_NAMESPACE:-false}
-export DELETE_OPERATOR_NAMESPACE=${DELETE_OPERATOR_NAMESPACE:-false}
-export DELETE_WATCHED_NAMESPACE=${DELETE_WATCHED_NAMESPACE:-false}
-export PGO_ADD_OS_CA_STORE=${PGO_ADD_OS_CA_STORE:-false}
-export NAMESPACE_MODE=${NAMESPACE_MODE:-dynamic}
-export POD_ANTI_AFFINITY=${POD_ANTI_AFFINITY:-preferred}
-export PGO_APISERVER_PORT=${PGO_APISERVER_PORT:-8443}
-export PGO_APISERVER_URL=${PGO_APISERVER_URL:-https://postgres-operator}
-export PGO_CLIENT_CERT_SECRET=${PGO_CLIENT_CERT_SECRET:-pgo.tls}
-export PGO_CLIENT_CONTAINER_INSTALL=${PGO_CLIENT_CONTAINER_INSTALL:-false}
-export PGO_CLIENT_INSTALL=${PGO_CLIENT_INSTALL:-true}
-export PGO_CLUSTER_ADMIN=${PGO_CLUSTER_ADMIN:-false}
-export PGO_DISABLE_EVENTING=${PGO_DISABLE_EVENTING:-false}
-export PGO_DISABLE_TLS=${PGO_DISABLE_TLS:-false}
-export PGO_TLS_NO_VERIFY=${PGO_TLS_NO_VERIFY:-$PGO_DISABLE_TLS}
-export SERVICE_TYPE=${SERVICE_TYPE:-ClusterIP}
-
-# Metrics Defaults
-export METRICS_NAMESPACE=${METRICS_NAMESPACE:-pgo}
-
-# Installer Settings
 export ANSIBLE_CONFIG="/ansible/ansible.cfg"
 export DEPLOY_ACTION=${DEPLOY_ACTION:-install}
 export HOME="/tmp"
-export KUBERNETES_IN_CLUSTER=${KUBERNETES_IN_CLUSTER:-true}
 
-cat /inventory_template | envsubst > /tmp/inventory
-/usr/bin/env ansible-playbook -i /tmp/inventory --tags=$DEPLOY_ACTION /ansible/main.yml
+/usr/bin/env ansible-playbook \
+    -i /inventory.yaml \
```

Comment:
This could also work. With this change there would only be one `inventory.yaml` file, instead of having one for the `pgo-deployer` and one for ansible. The only difference is that `kubernetes_in_cluster` needs to be set somewhere for ansible to work in the pgo-deployer image. The `inventory.yaml` file for ansible is baked into the image along with the other roles. This can be used and we can set `kubernetes_in_cluster` at the command line.

```suggestion
    -i /ansible/inventory.yaml \
    --extra-vars "kubernetes_in_cluster=true" \
```

Reply:
@jmckulk As long as that extra variable is encapsulated so that the user does not have to do anything I'm ok with that approach.

Reply:
Yes this flag would be encapsulated in the `pgo-deployer` and it wouldn't change how users interact with `kubectl`

Reply:
Then I'm ok with it :wink: